tests/analyzer_tech_debt.txt

```diff
@@ -42,6 +42,7 @@
 02003_WithMergeableStateAfterAggregationAndLimit_LIMIT_BY_LIMIT_OFFSET
 02404_memory_bound_merging
 02725_agg_projection_resprect_PK
+02763_row_policy_storage_merge_alias
```

Comment:
Why?

Reply:
Basically because of 
```
$ grep merge_alias analyzer_tech_debt.txt 
01214_test_storage_merge_aliases_with_where
01925_test_storage_merge_aliases
```
Aliases in Merge do not work if analyzer enabled.